### PR TITLE
Allow other cells in colonies to use slime jets

### PR DESCRIPTION
**Pull request by Oliveriver**
**Brief Description of What This PR Does**

Fixes a bug where only the lead cell would use slime jets in multicellular.

**Related Issues**

N/A

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/early_multicellular_stage/Microbe.Multicellular.cs [hunk 1]
```diff
@@ -131,6 +131,23 @@ public void BecomeFullyGrownMulticellularColony()
         }
     }
 
+    /// <summary>
+    ///   Perform an action for all members of this cell's colony other than this cell.
+    /// </summary>
+    public void PerformForAllColonyMembers(Action<Microbe> action)
```
Comment (hhyyrylainen) [inline src/early_multicellular_stage/Microbe.Multicellular.cs:137]:
```suggestion
    public void PerformForOtherColonyMembers(Action<Microbe> action)
```

The name having "all" in it, but the comment then clarifying it doesn't actually mean "all" is pretty confusing so I'd recommend this rename.

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]

File: src/early_multicellular_stage/Microbe.Multicellular.cs (hunk 1 above)
Comment (hhyyrylainen) [reply src/early_multicellular_stage/Microbe.Multicellular.cs:137]: I fixed this myself and went with an even more verbose naming.

**Review by hhyyrylainen — APPROVED** [review]
Fixed one bug (if player cell didn't have jets but other members did have, this didn't work), and improved the code clarity for the "run on other colony members" part.

**Comment by hhyyrylainen** [comment]
I'll merge this unless @Oliveriver has something to say about my changes.

**Comment by Oliveriver** [comment]
Good changes, I didn’t realise I missed that bug. The method name is definitely unwieldy and I’m sure there’s a better alternative, but I don’t think it warrants too much work.

**Pull request merged**